docs/fundamentals/connection/tls.txt

```diff
@@ -0,0 +1,202 @@
+.. _laravel-tls:
+
+========================
+Enable and Configure TLS
```

Comment:
I'm beating a dead horse here, but it's bizarre to me that we're documenting this stuff on the Laravel library instead of in PHPLIB itself. Even then, this functionality is entirely contained within the extension (and libmongoc), but I understand that PHPLIB needs to fulfill the role of being the main docs for the PHP driver.

The only thing I would expect Laravel docs to cover would be how to specify these options in its own configuration (i.e. the mapping between Laravel config and the underlying driver options).

Is this the same content that's going to ultimately end up in PHPLIB as well? And is the official plan to have it duplicated?

Reply:
There is some content on setting SSL options in the existing PHP lib documentation - https://www.mongodb.com/docs/php-library/current/tutorial/connecting/#connecting-with-ssl-and-authentication

I guess the root issue here is that we are standardizing and creating content for the Laravel docs _before_ doing the same process for the PHPLIB docs. Since there is no comprehensive TLS documentation in the PHPLIB docs at the moment, we can cover this feature in this docs set, at least until we have a plan to standardize the PHP docs - maybe at that point, we can duplicate the content from here to there and replace the content in Laravel with a link to the PHP docs.

Reply:
@caitlindavey @ccho-mongodb If you have more context, feel free to comment!

Reply:
Side note: [Connecting with SSL and Authentication](https://www.mongodb.com/docs/php-library/current/tutorial/connecting/) is outdated, since it still uses the old `ssl` URI option instead of `tls`.

> maybe at that point, we can duplicate the content from here to there and replace the content in Laravel with a link to the PHP docs.

This is likely the best possible outcome I can hope for.